funtofem/optimization/openmdao_component.py

```diff
@@ -156,11 +160,10 @@ def compute(self, inputs, outputs):
     def compute_partials(self, inputs, partials):
         driver = self.options["driver"]
         model = driver.model
-        new_design = self.update_design(inputs, analysis=False)
+        self.update_design(inputs, analysis=False)
 
-        if new_design:
```

Comment:
Why aren't we still checking if it's a new design?

Reply:
I was having a problem getting it to run with the new_design mode. I will add an issue for this and address it later.